### PR TITLE
Fix bug in SchedulerCallback and write all LRs to metrics

### PR DESCRIPTION
**Pull request by and-kul**
## Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md)?
- [x] Did you check the code style? `catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).
- [x] Did you make sure to update the docs? We use Google format for all the methods and classes.
- [ ] Did you check the docs with `make check-docs`?
- [ ] Did you write any new necessary tests?
- [x] Did you check that your code passes the unit tests `pytest .` ? 
- [ ] Did you add your new functionality to the docs?
- [x] Did you update the [CHANGELOG](https://github.com/catalyst-team/catalyst/blob/master/CHANGELOG.md)?

## Description

1. Fix bug in SchedulerCallback, described in #989 
2. If you have many param_groups in optimizer then log LRs and momentums for each of them

## Related Issue

https://github.com/catalyst-team/catalyst/issues/989

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

**Comment by Scitator** [comment]
could you please check fix the codestyle?
`catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).

it would be also cool, if you could drop a minimal example in colab with "how now it working" example :)

**Comment by and-kul** [comment]
Codestyle is fixed

File: catalyst/utils/torch.py [hunk 1]
```diff
@@ -124,6 +124,27 @@ def get_optimizer_momentum(optimizer: Optimizer) -> float:
     return betas[0] if betas is not None else momentum
 
 
+def get_optimizer_momentum_list(
```
Comment (Scitator) [inline catalyst/utils/torch.py:127]: could you please also add it 
- to `__all__` in the bottom
- to import in the `catalyst/utils/__init__.py`
?

File: catalyst/callbacks/scheduler.py [hunk 2]
```diff
@@ -116,55 +119,89 @@ def _scheduler_step(
...
+        ]
+        momentum_list = get_optimizer_momentum_list(scheduler.optimizer)
+        return lr_list, momentum_list
 
-        return lr, momentum
-
-    def step_batch(self, runner: "IRunner") -> None:
-        """Update learning rate and momentum in runner.
+    def _update_lr_and_momentum_in_metrics_dict(
+        self, metrics_dict, lr_list, momentum_list
```
Comment (Scitator) [inline catalyst/callbacks/scheduler.py]: could you please add typings here?

File: catalyst/callbacks/scheduler.py [hunk 3]
```diff
@@ -116,55 +119,89 @@ def _scheduler_step(
...
+    ):
+        """Update learning rate and momentum in metrics_dict
 
         Args:
-            runner: current runner
+            metrics_dict (dict): batch_metrics or epoch_metrics
+            lr_list (List[float]): lr for each param group
+            momentum_list (List[float, None]): momentum for each param group
```
Comment (Scitator) [inline catalyst/callbacks/scheduler.py]: but remove them (typings) from the docs?

File: catalyst/utils/torch.py (hunk 1 above)
Comment (and-kul) [reply catalyst/utils/torch.py:127]: done

File: catalyst/callbacks/scheduler.py (hunk 2 above)
Comment (and-kul) [reply catalyst/callbacks/scheduler.py]: done

File: catalyst/callbacks/scheduler.py (hunk 3 above)
Comment (and-kul) [reply catalyst/callbacks/scheduler.py]: done

**Pull request merged**